src/Notifications/data/thunks.js

```diff
@@ -35,21 +35,23 @@ const normalizeNotificationCounts = ({ countByAppName, count, showNotificationsT
   };
 };
 
-const normalizeNotifications = ({ notifications }) => {
+const normalizeNotifications = (notifications) => {
   const newNotificationIds = notifications.map(notification => notification.id.toString());
   const notificationsKeyValuePair = notifications.reduce((acc, obj) => { acc[obj.id] = obj; return acc; }, {});
   return {
     newNotificationIds, notificationsKeyValuePair,
   };
 };
 
-export const fetchNotificationList = ({ appName, page, pageSize }) => (
+export const fetchNotificationList = ({ appName, page }) => (
   async (dispatch) => {
     try {
       dispatch(fetchNotificationRequest({ appName }));
-      const data = await getNotifications(appName, page, pageSize);
-      const normalisedData = normalizeNotifications((camelCaseObject(data)));
-      dispatch(fetchNotificationSuccess({ ...normalisedData, numPages: data.numPages, currentPage: data.currentPage }));
+      const data = await getNotificationsList(appName, page);
+      const normalisedData = normalizeNotifications((camelCaseObject(data.results)));
+      dispatch(fetchNotificationSuccess({
+        ...normalisedData, totalPages: data.num_pages, currentPage: data.current_page, nextPage: data.next,
```

Comment:
create a pagination object in normalizeNotifications. This object will have only the required fields. e.g
```
const pagination = {
 numPages: data.numPage,
 currentPage: data.current_page, 
 hasMorePages: !!data.next
}
```